index.html

```diff
@@ -369,7 +376,7 @@ <h4>Multikey</h4>
           </pre>
 
           <pre class="example nohighlight" title="Two public keys (P-256 and
-            Dilithium-2) encoded as Multikeys in a controller document">
+            ml-dsa-44) encoded as Multikeys in a controller document">
```

Comment:
```suggestion
            ML-DSA-44) encoded as Multikeys in a controller document">
```

Reply:
If you tell the me 2 bytes to put in front of SK and PK, I can update the PoC. Current situation:

**SK:**

```
{
	"Alice": {
		"keyring": {
			"mldsa44": "Xz/mYesZRXWoCZISAIw37dVDkvcAq3a096w+rfCzQCRGNB5upRAgJC9aTgxa0pm4LfADCtYommcXAR8QPvVqSev/FBEtvLLQDL0LzG61p9DInn4VSPupjVSoyyOhltqvR91yjrF3eFYTa5raqKbebbZ7m9fAntd/9429uKPBE1HTKIWItDEYQgHaxFDbNiQEBjLcgDChJkGhKELjQEYgBWHLEApRpI3BkEjbAC0Qtw2LJg4kx3BLFm2gRnGJNm4KM2VJBiICBgKgiCnABEojNIkIqESDNkViwmkLpYxZQAggiQ3JlGASmIAECECIwEGCQmBTtkjBNkULwwgcmSVIMiqZFhIcwDGjtAzBJmEclEERRoogSQgAR2XUGGkLwSELxnEiMgpIJHJbKJHktg3YNG4DIlDEFEVLwEQLtDCYhpBRBGLTFgQBAxHZxAFAxgHZMgwZSVLZRiggMFIKEHIDh5AEsgWiGEbIyDHbskSQAA6ZhCBQFlIgBIqQCGHAgpBQmIxhIiwjs2ABAygTQywCxIFawA2iEpELSS4RQGHhIi3TSEIEFmDJJFBLCJIYSUGBNhCkEgYCx2QIEzEisSGAkhFUtongJkIMmQkgsGkIRJFDKFCbwCDSxBGEFEQMSEoZIi5ZAEIhFoobKYJKMowZlyiMsGAMNoEIJSYbJUjEgkGEQgQJJFBgIGaQRIhguI1TlI0TEyIMAzIhwTAIuQkREUojF2SKJA7bFopLtiUbkUVIFFDIFg6aFClByECQRHASQQRZsEXhiIwEERELtzASuWxMME3MEHCKEiwLkCUkAQzkIAUkCE5SNnAkwwBcpgggN43gAIFUEkHQGFIKuVGEoI0JNIQQEk2DiIwDOASiNmwBECxEsoBLQkQapGRJSELRuIggo0USKG2JGEngyEwkFi7EFCailkkKFHKctAWaNI0YBUSMNAhYMGwjtITTBgFDIpFQKFEISQ7bEAXYxGQKMyFgJkZYACKIIAYEoQjjKGzCSClgNpBaBHBREgxLwCVKNE6TBohIRmICGYIIpYGkAmiMoowiCTKgkjEhkWRJMpDgliWYFAyApgnEJowjgWzTEE6aRAQLBkqRwESDIgJBFJBaBIRDEi7IQAyYNg7coEAjkU0MEwFkNkDbJggkoY3UkADhMmFbsikUKE0aBlEAtk3TQoKANjCZBGYjOS0INgLS2wTUIfC3D4hK4u1qvWzitDzqEs1EhJi520bDgRH6NYjLz3CccmkrwWsadYkKJkexZQB7+tCWDj7d72Q+lGozMmsieEYC0A5UNNrUZPPbQXZI/J2JIDdq9QDuSNvHbb6DDq9OgT3z/jR3v3qFzgVZn0Vc1THtY1gMBQfXQdE2UXPDYvd4zlgerfNZRlgSwIVWYwZ31Os42mm17lVwN9GlzCStrBxC+e8xwSzLAaMOpfidNUi7WX0CI846lZrAZj4GiaWPYe1Zfk/R4HJXHDM1DN0algTGr9FuCc6iO+o/EJyfEVL5C+Udka0AyKal7xNkC43v1VPKjrK/QEaffi/eArQcBYR9jdMs+Bq9sD1KvPL5X2lukmJkI8RxjZNLwZSsOqveEuDeHGqZr/PUYUWO8Eo6FAZNOKYNfuD3Be6jIIQophklB62W4MrUT95kDVWUWrHZtfVc3DFPqSbJcQC4Jiii3aU+Yo8jAvyDJYk6iCcB7e1hSubzABBYD2quNAKhiJ2RUxkS8VX5s9+SQlXjE6F+oqck1ncnMFSPX9A2D4n/82blKcWxEUVkszekblXk4kEvFLHe3sP2jQ/MJCCSBtl+3vG1mkPCWAHT6PHK49EuTTRJ1NFMb+AEQKPWnxEZj5kHhJBok1NWrZk8dBYEcWz/qD5HzHYwtFckNs6DN/vPiIoSktSl0XxSpCedhvwCV+cxPppPvkcQukhrv97vU0PhfAYX1LLNMVeCDsoCjjlsJfTwOzJKF1QdZStVl2CJFl48gWXUZiho2Q37Vi9GUpbJYrPhTcpjUZ132qHZ6dG1SjN+yq+pCDvnqCAGXT+4CLso6j1wqvLjfvVtekc0uXMCnk+lAlms9r0Ps1dWTlTYvAi9aK5f/bLu1QPYH2ryOccrePvZPBjb4XZU3DGqkvCrl4Mxk3eo0SOtNMORfqC7OSE4+npYOFuRbGbVXSbAPLZXzVoCiWlvY9PME3QBBOfj4MHSVIW65Tm9lYn2unHA/5VaxftFYXnvmL3jxBgb+XpCwn9crlKfGNZ15xGGYTAN9SOcvqFhuRPumdwaDTLk8uO46+o8u1u21ICroUTNlK19pEJvD0i32lHiqcFgXpHloG3Sm51YkrM/dvKwgDCVhyvFNIFRoz3TIUtP9yPSK4gAhmVcgROaN5uYAxarwI1vhR9K24wW1VtIxMU3ifGwhs2IeVeno1FiaTZFSTaljy2p8ntnQ8gQpT719MvwErmsgrTOJv9IPXwyT8qhm5IcPIHzv4UnDFYXRC5d39PaK+uXww3qjXfKFisCC6Uwl9IbVs5MSYbayBeW9/RmAfOCGSaT7NBPBINN1gcpdEq3/HSrGp8Prsfs3zvJsQm5rO4oYZotjXe7Ob7qTDl3pjw1ebYIyCpC1zKuVG/2GygTOlzkb/PUNZpXNGtNL/OisvLnjC7FBEPSyFKfg+Qv/+mGeByix1EaRC7UQmnq5sYou9Dl/V7C1vwqgCsJrtppwHnPrOuoBUEyZbcoEYhZdtZZtPDtnfqAsQBUL255aUzkI4GLGky/gjJ7cGpUyaUWewYe1E9Q8GZKsICziCofeu+z6GQ7lI33N86UP094uc0svQUBzf0zErjX2eN7G8jTgj/6K/23KVT3NHGv03YYZIUDMo9YRtyulcjY+1sMYatr3A+mDURCB1FoM0AfwW55rkG7HVEPCHHERhlTYQedNTkUgz5w3tik4/yMUh+5OhX+nEe1GJDkCi4/a4YkxYapggiErAG80wGJpAjCIBv7lhKsOROGp7SG06XD7N/DaQ2Ez3WiWT9Ohg0t0yrx72+6yPBG01XAtgJ5/k3uj3C72/9h48Hx9Y9PF7S9VQ5Yi7ajyCO71KsZ/E+hxIgAldcW/aQZQOK0ZbauU+3X7nNTlUJtGM45TMOORDw7O8uBrOU1fJA8YGM0M/7ANW+80P0+Jiv8Av59iLCPoGk1jnIPzMNXpwrZnGy3A5vSAULwE1GM6eWfPv8xBbNKrLEWbu6v0QEU2IPZ28wOoqqWYZR74b7fVYOAwXKKVALeA2yybI7Hp6XX72HZ7r7Yb9NDFavPhkHSZivxmoSM4nsfQ4Jf+tq1oWxeuYAWe5W6e80stiTGGwPqlKBUS/ybL17jbbSlYuDdNrV2bccz6unAgWdMUC47i7exXXl4TC+9Bh/pFhfZr1c0dnAi343layH7yQw8DfUNlFjddaoT8zAohgLzMA=="
		}
	}
}
```

**PK:**

```
{
   "Alice": {
      "mldsa44_public_key": "Xz/mYesZRXWoCZISAIw37dVDkvcAq3a096w+rfCzQCRQBxu0y9zKyyEdjQIi19CT+UwKkdzVZTmLL4m5iZxTcmN4zMIBnVy/cbPw1XV41FbX4lWxPDvWqiFCUsDEiOeXG2BJO0VNyWuccj7WBsKxFk1Q7N15IwaTiiMdvJVzJ8SaffASZ+f3vjfsQEGpnoZNBLG1tAdJjmxY3I4/c+/KBxI3cLkt3Al7oyUAoaYBaS5qRo23y7EE2cpi7R3SLUJ2bDbpJCEJRlriB/GFdVmGJ5B35tOTvqghzzgT0yFiuAVJ2ekawrEz7JAItrC2vNk6OnhxTUk4hq4q4SkG3Tdiw9XvE7fIsZni/uneFq2wDNR6sqySOWCsMYL5s7shZnMHP7HkK84jjVin3euz8i8LkLw02J3kYGDPAsBZ97wkNrt6/zpPW7TpVbHthxNi9aBM35vdqpBWhOpzPJSvrc7iUDRfT4mnR6a4OvRsQG/O5ez3W0ka79x2wgfjwrEXF9GEK4QgE0XZaHE16UcHeGvaEiGMiFxWfGcg/PlBEbxlJzId4fGa04CJU5DS02HV8s4aPMJ4FMqX+mmjtQc6xfOqtvmlTu6fsdgFwzZ+Yfm2TL5dGEzVr5FxxERDCLwCoqEsMyxzJzO/2jBFJOEk2dhN/tFLEqmNOyJmvOC3jOipQ6aUbyqpzZc+OMyhHY139PK2LaEjihwlPOBrKuu74uwozE5Pp0tlgvH9HSMbAnesXtk5STw4jnssKW4+Jgi/Dm/OBXz1QFGdIplq7EcLhuiSBKrFMMHFCNCIaIeqwkC7yH2AQAxAkY9TWH2wn0QuGCn/vZodjTesBjyCQotNoGZeMWxmqHVEhyot2/NYL8mbM4euX88tJsONWFh/OK8P27M+6i6GBwteXo2hBHEwTT6koAKMj0Ce1I0hIngcp9ki0FxJIJkB2yrLt8JYkWmtK1fbi4bvUjbGF7tKrqpr2gM45BXzZqFe9GeqKRwZhJDJ1GEKR5ysK874tV6rOoARudSBhSX9c43NIjMBqGlg3ROc4GJ2CB7l1+SrHwtDQSDcOn5vId0/TM8J1su+KFYJAYSsqX5HoNlAe/aNLRcDbfXVJKARhWVcfgk841nEFgVbdoVNFYAODmqg7nil74xfjV3YyDr15mSqwyeOwY1+fSFwN9yJK26wQ+VohQBZTb7y/jnvQJLkVxbVrQUjVtQdcRW3HiIflAPijfX+arpntb9RO4qwuoRuHAKeNu4tYSPtJFVXIOZogiWoowgNHdiSxadFpstc8G7uFyL0swA7KAZ7EKde/Jtq5LJo60GuJ3pWCsdNfO6jRyV+qNGZzVD2RQjgYOzQnYNoxF2d1ngMk0nzUKcGtFodTiv5lu8/1vfP7NJ1Q6kEsPnHYTNNcCX2wynMrra9zwgpsGuuAlNyZ2l7NmSMC484t1xdmFzMj33C6YG1fnWwyN8K7CbVSDRAF00c0uqOeBef1vDHAeXS0DByksST02jx4zXSyyjW1+wpBCpn4Y3lpSES/26ojPR3E4yM6K0cnnvuF5RRAfAMXifTbHV9/Sy8hKPdRX4/1JvdfjJe9kVE32du72ErfZge9YriR9ARUzLI2eZF3sVFq1miQFB57CrHgiyWfF0R14P1hL5k48k4eRYdTerhm0IgRxHg8XrqVxHthsxRIYixgt9avt6555zIRXA1izJL5w9tejJscQLns+6MiRt0BVzFkjqxzcSaQFkYzfWSfpE0PFxkZw=="
   }
}
```